go.mod

```diff
@@ -51,16 +51,20 @@ replace (
 
 require (
 	github.com/containerd/containerd v1.3.6
-	github.com/ease-lab/vhive/cri v0.0.0-00010101000000-000000000000
-	github.com/ease-lab/vhive/ctriface v0.0.0-00010101000000-000000000000
-	github.com/ease-lab/vhive/examples/protobuf/helloworld v0.0.0-00010101000000-000000000000
-	github.com/ease-lab/vhive/metrics v0.0.0-00010101000000-000000000000
-	github.com/ease-lab/vhive/proto v0.0.0-00010101000000-000000000000
-	github.com/golang/protobuf v1.3.5 // indirect
+	github.com/ease-lab/vhive/cri v0.0.0-20201130191325-566327025d78
```

Comment:
Lines 54-59 are stale, please delete them. in fact, revert this file to upstream/master and then run `go build` so that golang regerates go.mod again.

Reply:
Done

Reply:
nope, still wrong. the vhive ones should be like these:
```
	github.com/ease-lab/vhive/cri v0.0.0-00010101000000-000000000000
	github.com/ease-lab/vhive/ctriface v0.0.0-00010101000000-000000000000
	github.com/ease-lab/vhive/examples/protobuf/helloworld v0.0.0-00010101000000-000000000000
	github.com/ease-lab/vhive/metrics v0.0.0-00010101000000-000000000000
	github.com/ease-lab/vhive/proto v0.0.0-00010101000000-000000000000
	```
you need to add `replace` statements. Please copy them from the top module's go.mod